### PR TITLE
Fix parsing of flags

### PR DESCRIPTION
**Pull request by dveeden**
`rippled --help` wasn't working for me. This makes it actually parse and use commandline flags

### CODE REVIEW

File: rippled.cc [hunk 1]
```diff
@@ -43,6 +43,7 @@ void sighandler(int signo) {
 }
 
 int main(int argc, char **argv) {
+  gflags::ParseCommandLineFlags(&argc, &argv, true);
```
Comment (pivanof) [inline rippled.cc]: Can you put this into mysql_ripple::Init()? I think it will be more appropriate to keep all initialization there.

Comment (dveeden) [reply rippled.cc]: Yes I agree

File: external_libs.BUILD.bazel [hunk 2]
```diff
@@ -44,7 +44,7 @@ cc_library(
         "mysql/typelib.h",
     ],
     linkopts = [
-        "-l:libmariadbclient.a",
+        "-l:/opt/mysql/10.1.37/lib/libmysqlclient.a",
```
Comment (pivanof) [inline external_libs.BUILD.bazel]: This is not related (and doesn't exist on my machine ;) ).

Comment (dveeden) [reply external_libs.BUILD.bazel]: Sorry for that, I should have been more careful.
